DataSet.rst

```diff
@@ -456,39 +480,48 @@ DataSet External Resource MetaTypes and File Extensions
 | ReferenceSet          | PacBio.ReferenceFile.ReferenceFastaFile        | .fasta                    |
 +-----------------------+------------------------------------------------+---------------------------+
 
-SubreadSet Special Purpose ExternalResources:
-    - PacBio.SubreadFile.ScrapsBamFile
-    - PacBio.SubreadFile.HqRegionBamFile
-    - PacBio.SubreadFile.HqScrapsBamFile
-    - PacBio.SubreadFile.LqRegionBamFile
-    - PacBio.SubreadFile.LqScrapsBamFile
-    - PacBio.SubreadFile.PolymeraseBamFile
-    - PacBio.SubreadFile.PolymeraseScrapsBamFile
-    - PacBio.SubreadFile.ChipStatsFile (.sts.xml)
-    - PacBio.SubreadFile.ChipStatsH5File (.sts.h5)
-    - PacBio.SubreadFile.AdapterFastaFile
-    - PacBio.SubreadFile.ControlFastaFile
-
-Bam-Related Special Purpose ExternalResources:
-    - PacBio.Index.BamIndex (.bam.bai)
-    - PacBio.Index.PacBioIndex (.bam.pbi)
-
-Fasta-Related Special Purpose ExternalResources:
-    - PacBio.Index.SamIndex (.fasta.fai)
-    - PacBio.Index.SaWriterIndex (.fasta.sa)
-    - PacBio.Index.Indexer (.fasta.index)
-    - PacBio.Index.FastaContigIndex (.fasta.contig.index)
-    - PacBio.GmapDB.GmapDBSummary (a file indicating the location of a gmap db)
+Some ``ExternalResource``'s themselves contain associated ``ExternalResources``'s, for example the
+indices associated with FASTA files. These associated files are nested within the primary ``ExternalResource``
+element to denote their subsidiary nature.
```

Comment:
It's not just a matter of type though, it's an issue of where the nesting occurs - we wouldn't want this either::

  <SubreadSet>
     <ExternalResources>
       <BarcodeSet>...</BarcodeSet>
     </ExternalResources>
  </SubreadSet>

or (this is probably closer to a real-world example)::

  <SubreadSet>
     <ExternalResources>
       <ExternalResource MetaType="PacBio.DataSet.BarcodeSet">...</ExternalResource>
     </ExternalResources>
  </SubreadSet>

In either case it doesn't matter what the exact dataset types are, same or different.   On the other hand, this is okay::

  <SubreadSet>
    <ExternalResources>
      <ExternalResource MetaType="PacBio.SubreadFile.SubreadBamFile">
        <ExternalResources>
          <ExternalResource MetaType="PacBio.DataSet.BarcodeSet"/>
        </ExternalResources>
      </ExternalResource>
    </ExternalResources>
  </SubreadSet>